src/pages/about.tsx

```diff
@@ -39,30 +36,27 @@ const PortfolioItemsList = () => (
   </ul>
 )
 
-const About = ({ location }: { location: any }) => (
-  <Layout location={location}>
-    <div className="bg-near-white lh-copy pa3-ns pv4 w-100">
-      <Helmet title={`About | jesses.io`} />
-      <StyledPanel className="mt2 f3">
-        <StyledLink className="alegreya-sans f2 fw5 w-100">
-          Get in touch
-        </StyledLink>
-        <div className="center flex items-center justify-center">
-          <img src={Logo} style={{ maxHeight: 200, marginBottom: 0 }} />
-        </div>
-        <p className="lato f4 fw4 mt5">
-          You can check out a sampling of my publications below, or scope out my
-          C.V. to learn more about what I've been up to.
-        </p>
-        <p className="lato f4 fw4">
-          Feel free to hit me up via whichever communication platform you prefer
-          (
-          <ContactInfoList />, etc.) if you'd like to chat.
-        </p>
-        <PortfolioItemsList />
-      </StyledPanel>
-    </div>
-  </Layout>
+const About = () => (
+  <div className="bg-near-white lh-copy pa3-ns pv4 w-100">
+    <Helmet title={'About | jesses.io'} />
+    <StyledPanel className="mt2 f3">
+      <StyledLink className="alegreya-sans f2 fw5 w-100">
+        Get in touch
+      </StyledLink>
+      <div className="center flex items-center justify-center">
+        <img src={Logo} style={{ maxHeight: 200, marginBottom: 0 }} />
+      </div>
+      <p className="f4 fw4 mt5">
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 12 spaces but found 6. (@typescript-eslint/indent)](https://app.codacy.com/app/jdstuart/jesses.io/pullRequest?prid=4576429)